### DISCUSSION TITLE
How to install GeoChemFoam-5.0 from source code?

### DISCUSSION THREAD

**Discussion by Zhao-yijin** [GeoChemFoam/GeoChemFoam#8] [Q&A] [answered]
Dear Professor Hannah,
When I installed  GeoChemFoam-5.0 based on your tutorial (https://github.com/GeoChemFoam/GeoChemFoam/wiki/GeoChemFoam-Native-version:-Install-GeoChemFoam-from-source-code), I found some problems that confused me:
![Install5 0](https://user-images.githubusercontent.com/67924068/237011830-3108c55f-7dc1-4fb1-95b5-d1778e1c568d.jpg)
According to the"Prerequisite", GeoChemFoam-5.0 uses the ESI OpenCFD Release OpenFOAM-v2212. However, when it comes to "Set-up the bashrc script", the bashrc file gives the path to the foam-extend-4.0 directory, instead of the OpenFOAM-v2212 directory. 

So before I install GeoChemFoam-5.0, should I download foam-extend-4.0 like  install GeoChemFoam-4.8, or download OpenFOAM-v2212 instead of foam-extend-4.0?

**Comment by julienmaes** [comment, marked as answer]
Dear Zhao, you're a bit faster than us! We are in the process of releasing version 5.0 and we are modifying the website accordingly. Foam-extend-4.0 was needed for version 4.8 but not anymore, only openfoam2212.

Keep an eye on the website, every page will be updated in the next couple of weeks.

Regards,

Dr Julien Maes

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: Zhao-yijin ***@***.***>
Sent: Tuesday, May 9, 2023 10:22:40 AM
To: GeoChemFoam/GeoChemFoam ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [GeoChemFoam/GeoChemFoam] How to install GeoChemFoam-5.0 from source code? (Discussion #8)

****************************************************************
Caution: This email originated from a sender outside Heriot-Watt University.
Do not follow links or open attachments if you doubt the authenticity of the sender or the content.
****************************************************************


Dear Professor Hannah,
When I installed GeoChemFoam-5.0 based on your tutorial (https://github.com/GeoChemFoam/GeoChemFoam/wiki/GeoChemFoam-Native-version:-Install-GeoChemFoam-from-source-code), I found some problems that confused me:
[Install5 0]<https://user-images.githubusercontent.com/67924068/237011830-3108c55f-7dc1-4fb1-95b5-d1778e1c568d.jpg>
According to the"Prerequisite", GeoChemFoam-5.0 uses the ESI OpenCFD Release OpenFOAM-v2212. However, when it comes to "Set-up the bashrc script", the bashrc file gives the path to the foam-extend-4.0 directory, instead of the OpenFOAM-v2212 directory.

So before I install GeoChemFoam-5.0, should I download foam-extend-4.0 like install GeoChemFoam-4.8, or download OpenFOAM-v2212 instead of foam-extend-4.0?

—
Reply to this email directly, view it on GitHub<https://github.com/GeoChemFoam/GeoChemFoam/discussions/8>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AEINXJSXYQPFM4GKBX46KJDXFIEGBANCNFSM6AAAAAAX3ALDBA>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

________________________________

Founded in 1821, Heriot-Watt is a leader in ideas and solutions. With campuses and students across the entire globe we span the world, delivering innovation and educational excellence in business, engineering, design and the physical, social and life sciences. This email is generated from the Heriot-Watt University Group, which includes:

  1.  Heriot-Watt University, a Scottish charity registered under number SC000278
  2.  Heriot- Watt Services Limited (Oriam), Scotland's national performance centre for sport. Heriot-Watt Services Limited is a private limited company registered is Scotland with registered number SC271030 and registered office at Research & Enterprise Services Heriot-Watt University, Riccarton, Edinburgh, EH14 4AS.

The contents (including any attachments) are confidential. If you are not the intended recipient of this e-mail, any disclosure, copying, distribution or use of its contents is strictly prohibited, and you should please notify the sender immediately and then delete it (including any attachments) from your system.

**Reply by Zhao-yijin** [reply]
Thank you for your prompt reply! Looking forward to the release of version 5.0!

**Comment by GeoChemFoam** [comment]
Hi,

We haven’t officially released 5.0 yet as we are still updating the tutorials. Please be patient and we will update them in the next few weeks.

HM

On May 9, 2023, at 11:58, julienmaes ***@***.***> wrote:

﻿
****************************************************************
Caution: This email originated from a sender outside Heriot-Watt University.
Do not follow links or open attachments if you doubt the authenticity of the sender or the content.
****************************************************************


Dear Zhao, you're a bit faster than us! We are in the process of releasing version 5.0 and we are modifying the website accordingly. Foam-extend-4.0 was needed for version 4.8 but not anymore, only openfoam2212.

Keep an eye on the website, every page will be updated in the next couple of weeks.

Regards,

Dr Julien Maes

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: Zhao-yijin ***@***.***>
Sent: Tuesday, May 9, 2023 10:22:40 AM
To: GeoChemFoam/GeoChemFoam ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [GeoChemFoam/GeoChemFoam] How to install GeoChemFoam-5.0 from source code? (Discussion #8)

****************************************************************
Caution: This email originated from a sender outside Heriot-Watt University.
Do not follow links or open attachments if you doubt the authenticity of the sender or the content.
****************************************************************


Dear Professor Hannah,
When I installed GeoChemFoam-5.0 based on your tutorial (https://github.com/GeoChemFoam/GeoChemFoam/wiki/GeoChemFoam-Native-version:-Install-GeoChemFoam-from-source-code), I found some problems that confused me:
[Install5 0]<https://user-images.githubusercontent.com/67924068/237011830-3108c55f-7dc1-4fb1-95b5-d1778e1c568d.jpg>
According to the"Prerequisite", GeoChemFoam-5.0 uses the ESI OpenCFD Release OpenFOAM-v2212. However, when it comes to "Set-up the bashrc script", the bashrc file gives the path to the foam-extend-4.0 directory, instead of the OpenFOAM-v2212 directory.

So before I install GeoChemFoam-5.0, should I download foam-extend-4.0 like install GeoChemFoam-4.8, or download OpenFOAM-v2212 instead of foam-extend-4.0?

—
Reply to this email directly, view it on GitHub<https://github.com/GeoChemFoam/GeoChemFoam/discussions/8>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AEINXJSXYQPFM4GKBX46KJDXFIEGBANCNFSM6AAAAAAX3ALDBA>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

________________________________

Founded in 1821, Heriot-Watt is a leader in ideas and solutions. With campuses and students across the entire globe we span the world, delivering innovation and educational excellence in business, engineering, design and the physical, social and life sciences. This email is generated from the Heriot-Watt University Group, which includes:

1. Heriot-Watt University, a Scottish charity registered under number SC000278
2. Heriot- Watt Services Limited (Oriam), Scotland's national performance centre for sport. Heriot-Watt Services Limited is a private limited company registered is Scotland with registered number SC271030 and registered office at Research & Enterprise Services Heriot-Watt University, Riccarton, Edinburgh, EH14 4AS.

The contents (including any attachments) are confidential. If you are not the intended recipient of this e-mail, any disclosure, copying, distribution or use of its contents is strictly prohibited, and you should please notify the sender immediately and then delete it (including any attachments) from your system.

**Reply by Zhao-yijin** [reply]
Thank you for your prompt reply! Looking forward to the release of version 5.0!
